infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryDao.kt

```diff
@@ -681,6 +680,10 @@ class GeometryDao @Autowired constructor(
         headerCache.putAll(headers)
     }
 
+    fun getPlanHeader(planId: IntId<GeometryPlan>) = getPlanHeader(fetchPlanVersion(planId))
```

Comment:
Itse planien hakuja ei voi koostaa näin dao:ssa koska ne tulee edelleen springin cachesta joten varsinainen haku pitää tulla dao:n ulkopuolelta. Headerit kuitenkin voi ja niitä kysellään paljon enemmän.